framework/test/mocha/unit/modules/http_api/controllers/node.js

```diff
@@ -194,6 +196,53 @@ describe('node/api', () => {
 		});
 	});
 
+	describe('_networkHeight', () => {
+		const MAJORITY_HEIGHT = 456;
+
+		let networkHeight;
+		describe('When there are set of majority peers with same height', () => {
+			beforeEach(async () => {
+				const defaultPeers = [
```

Comment:
Can we test more cases? (1 peer only, another height with count same as the majority, majority height way lower than other non-majority heights, etc.)